modules/programs/neovim.nix

```diff
@@ -341,9 +344,13 @@ in {
 
     neovimConfig = pkgs.neovimUtils.makeNeovimConfig {
       inherit (cfg) extraPython3Packages withPython3 withRuby viAlias vimAlias;
-      withNodeJs = cfg.withNodeJs || cfg.coc.enable;
       plugins = map suppressNotVimlConfig pluginsNormalized;
       customRC = cfg.extraConfig;
+      withNodeJs = if (cfg.withNodeJs == true || cfg.coc.enable
```

Comment:
or you know just use the boolean `cfg.withNodeJs == true || cfg.coc.enable  && cfg.withNodeJs != null)`, . Actually I would add parentheses for clarity `cfg.withNodeJs == true || (cfg.coc.enable        && cfg.withNodeJs != null))`